webpack/assets/javascripts/foreman_tools.js

```diff
@@ -35,3 +35,11 @@ export function activateDatatables() {
     });
   });
 }
+
+export function UserInheritedRolesUpdater(id) {
+  $("#roles_tab li[data-id = '" + id + "']").show();
+  $("#roles_tab li[data-id != '" + id + "']").hide();
```

Comment:
would it be better to first hide all `li`s and then show the right one? that way you wouldn't have to evaluate `!= id` for each `li` I think

something like

```
$("#roles_tab li]").hide();
$("#roles_tab li[data-id = '" + id + "']").show();
```